migtests/tests/import-csv-file-test/run-import-file-test

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
Can't we just add multiple invocations of `import data file` in the existing test case itself?